### ISSUE TITLE
Improve naming consistency

### ISSUE THREAD

**Issue by bsherman** [gerblesh/ublue-update#11]
A while back there was a big push to cleanup naming consistency in ublue RPMs.  

Seems like this could benefit from a similar effort.

I'd probably pick one name/phrase and stick to that.

It seems that `ublue-update` is the most commonly used so I'll use that as an example:

Changes I'd suggest:
- rename `update-ublue` script to `ublue-update`
- rename `/etc/update.d` to `/etc/ublue-update.d` (and also `ublue-update.conf` in that dir)
- rename `ublue-updater.spec` to `ublue-update.spec` (and the related resulting RPM, the name, etc in spec file)
- basically all the `ublue-updater` files and references drop the trailing `r`

This is not exhaustive but it hopefully communicates the idea.

It will feel cleaner and easier to discover relevant conf if the binary(script), systemd units, config files, paths, are all named consistently.

**Comment by gerblesh** [comment]
Fixed
